ui-tests/cypress/support/command/validation-commands.js

```diff
@@ -6,8 +6,9 @@ Cypress.Commands.add('validateListTexts', (list, values) => {
   }
 });
 
-Cypress.Commands.add('validateUrlPathname', pathname => {
-  cy.location('pathname').should('eq', Cypress.config('basePath') + pathname);
+Cypress.Commands.add('validateUrlPathname', (pathname, options = {portalUI: false}) => {
+  if (options.portalUI === true) cy.location('pathname').should('eq', Cypress.config('portalUIPath') + pathname);
```

Comment:
`options` cannot be null, neither can `portalUI`; since `options.portalUI` is a boolean, there's no need to compare it to true.
As the only difference is in the prepended path, you could create a const based on the value of the `options` and use that in the `cy` command